### ISSUE TITLE
Resume during the demo session

### ISSUE THREAD

**Issue by favrei** [2018-summer-DL-training-program/Lab-3-CVAE-InfoGAN#1]
Hello TAs,

There is no resume or reload of the models in the reference code of Lab3(CVAE).
I don't understand how will the live demo go with this.
Do we need to implement these functions on our own?

![image](https://user-images.githubusercontent.com/17000685/45137039-8c05a580-b1d9-11e8-9efd-7a96e684ffba.png)

Thank You,

**Comment by chunchih** [comment]
@killer9619 Yes, you need to implement the function to reload the model. With my inference code provided this week, I think it's not hard for you to do that.

**Comment by favrei** [comment]
Thank you for the reply.
